### PR TITLE
Bump subxt from 0.25.0 to 0.28.0 and adapt to new API

### PR DESCRIPTION
**Pull request by smiasojed**

### CODE REVIEW

File: src/wasm/xts.rs [hunk 1]
```diff
@@ -34,45 +31,55 @@ impl ContractsApi {
...
+        let bytes: Bytes = self
+            .client
+            .rpc()
+            .request("state_call", params)
+            .await
+            .unwrap_or_else(|err| {
+                panic!("error on ws request `contracts_instantiate`: {err:?}");
+            });
+        Decode::decode(&mut bytes.as_ref())
+            .unwrap_or_else(|err| panic!("decoding ContractInstantiateResult failed: {err}"))
```
Comment (ascjones) [inline src/wasm/xts.rs:56]: Similar code block to the dry run call later on, so could be factored out. But only used in two places so marginal benefit only. Up to you.

File: src/wasm/xts.rs [hunk 2]
```diff
@@ -89,20 +96,29 @@ impl ContractsApi {
...
+        let bytes: Bytes = self
+            .client
+            .rpc()
+            .request("state_call", params)
+            .await
+            .unwrap_or_else(|err| {
+                panic!("error on ws request `contracts_call`: {err:?}");
+            });
+        let result: ContractExecResult<Balance> = Decode::decode(&mut bytes.as_ref())
+            .unwrap_or_else(|err| panic!("decoding ContractExecResult failed: {err}"));
```
Comment (ascjones) [inline src/wasm/xts.rs:120]: This block

**Review by ascjones — APPROVED** [review]
LGTM!

**Comment by ascjones** [comment]
Also CI needs fixing

File: src/wasm/xts.rs (hunk 2 above)
Comment (smiasojed) [reply src/wasm/xts.rs:120]: I will update it, thanks

**Comment by ascjones** [comment]
> > Also CI needs fixing
> 
> Shall I do it? I am not sure where the issue is, looks like related to ink! examples

Ah yes I see, because the `examples` dir has been renamed to `integration-tests`. Please go ahead and change that.

**Pull request merged**